tests/improver-nbhood/02-basic.bats

```diff
@@ -46,3 +46,20 @@
   rm "$TEST_DIR/output.nc"
   rmdir "$TEST_DIR"
 }
+
+@test "nbhood 'square' --radius-in-km=20 input output" {
```

Comment:
When tests fail, it says how many of the subtests failed but not which of the subtests have failed. So it will be slightly harder to debug. I'll move it to a separate file.